rules/keep-tasks-handy-for-calls/rule.md

```diff
@@ -0,0 +1,32 @@
+---
+type: rule
+title: Do you always keep your task list handy for when the boss calls?
```

Comment:
Hi @Tanya-Leahy 

1. Please talk to Uly if the word "Manager" would suit here better than "boss". I am not sure about it...

Reply:
Done - Spoke with Uly and changed rule reference "boss" to "manager".